addons/hr_expense/__manifest__.py

```diff
@@ -36,10 +36,9 @@
         'wizard/hr_expense_sheet_register_payment.xml',
         'views/hr_expense_views.xml',
         'security/ir_rule.xml',
-        'report/report_expense_sheet.xml',
-        'views/hr_dashboard.xml',
-        'views/account_journal_dashboard.xml',
```

Comment:
As written in the saas-11.2 pad, you removed ''views/account_journal_dashboard.xml'' that is no longer loaded by the manifest.